script.java

```diff
@@ -0,0 +1,151 @@
+//usr/bin/env jbang "$0" "$@" ; exit $?
```

Comment:
`script.java` is a little bit non-descriptive. Something like `apply-revapi-suggestions.java` would IMHO be better so that people don't have to guess at the purpose of this script.